.github/workflows/ci.yml

```diff
@@ -234,25 +234,26 @@ jobs:
           name: output
           path: _output/**
 
-      - name: Login to Docker
+      - name: Login to Marketplace
```

Comment:
```suggestion
      - name: Login to Upbound
```

Reply:
suggesting r/Marketplace/Upbound on this file; feels more precise as "Marketplace" could be ambiguous

Reply:
What about Upbound Marketplace?  More specific and more branded.